### PR TITLE
website: linux installation docs

### LINKED ISSUE #337: provide installation guide for Linux users
Primarily install and test in a Fedora environment and publish the documentation on the website

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### CODE REVIEW

**Comment by deekay2310** [comment]
Screen Shot 2022-07-26 at 12 43 23 PM
Screen Shot 2022-07-26 at 12 43 33 PM

**Comment by deekay2310** [comment]
added information on having flatpak set up

File: website/docs/Installation/linux-install.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 5
+---
+
+# Linux
+
+## Installing Podman Engine
+
+In order to use Podman Desktop, you need to have the latest version of Podman Engine. Fedora-CoreOS comes with Podman Engine. All you need to do is make sure that it is the latest version available.
```
Comment (benoitf) [inline website/docs/Installation/linux-install.md]:
```suggestion
In order to use Podman Desktop, you need to have a 4.x version of Podman Engine. For example Fedora 36 comes with Podman Engine 4.x. All you need to do is make sure that the linux distribution is using a 4.x version 
```

File: website/docs/Installation/linux-install.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+For other Linux Distributions, click [here](https://podman.io/getting-started/installation#installing-on-linux) for a guide to install Podman Engine for different linux Distributions.
+
+Now that you have Podman machine up and running, you can check out the [Downloads](/downloads/linux) section of this website to download either the [.flatpak](https://flatpak.org/setup/Fedora) file or the zip file depending on your preference.
+
+## Installing Podman Desktop
+
+### 1. Using [flatpak](https://flatpak.org/setup/)
+
+Before you follow along, make sure flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up flatpak.
```
Comment (benoitf) [inline website/docs/Installation/linux-install.md]:
```suggestion
Before you follow along, make sure Flathub is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up flatpak.
```

File: website/docs/Installation/linux-install.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+cd Downloads
+```
+
+Run the following command to install the flatpak application,
+
+```sh
+flatpak install <name_of_the_flatpak_file> 
+```
+
+In order to start the application, you need to know the Flatpak identifier for the application. In our case, that is ```com.github.containers.desktop```
```
Comment (benoitf) [inline website/docs/Installation/linux-install.md]:
```suggestion
In order to start the application, you need to know the Flatpak identifier for the application. In our case, that is `io.podman_desktop.PodmanDesktop` starting 0.0.6 and `com.github.containers.desktop` for previous versions
```

File: website/docs/Installation/linux-install.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Run the following command to start the application,
+
+```sh
+flatpak run com.github.containers.desktop
```
Comment (benoitf) [inline website/docs/Installation/linux-install.md]:
```suggestion
flatpak run `io.podman_desktop.PodmanDesktop`
```

File: website/docs/Installation/linux-install.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+You can read more about running Flatpak applications [here](https://docs.flatpak.org/en/latest/using-flatpak.html).
+
+### 2. Using zip file
+
+Download the zip file and extract the folder.
+
+Navigate within the folder and double-click on the ```podman-desktop``` executable file. This should start the application for you.
```
Comment (benoitf) [inline website/docs/Installation/linux-install.md]:
```suggestion
Navigate within the folder and double-click on the `podman-desktop` executable file. This should start the application for you.
```

**Comment by deekay2310** [comment]
@benoitf should I go ahead and merge?

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
@deekay2310 yes sorry I forgot to hit the review button few hours ago

**Pull request merged**